src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -12,6 +12,15 @@
     <WriteLinesToFile File="$(OutputPath)/.version"
                       Lines="$(GitCommitHash);$(Version);$(Rid)"
                       Overwrite="true" />
+
+	<!-- Development env and CI loads DotnetFiles type from the following location
+	      The resolution of the product version (https://github.com/dotnet/sdk/blob/main/src/Cli/Microsoft.DotNet.Cli.Utils/DotnetFiles.cs#L21)
+		  then need the version file there as well. -->
+    <WriteLinesToFile File="$(BaseOutputPath)$(Configuration)/.version"
```

Comment:
You could just do a copy, that way if the file changes we don't need to modify it in multiple places

```xml
<Copy SourceFiles="$(OutputPath)/.version" DestinationFiles="$(BaseOutputPath)$(Configuration)/.version" SkipUnchangedFiles="true" />
```